### DISCUSSION TITLE
twitch integration

### DISCUSSION THREAD

**Discussion by mirobane** [ausbitbank/stable-diffusion-discord-bot#43] [Ideas]
hi !

could be a good idea to link twitch chanels point to the bot for use as currency ?

scenario :

based using discord twitch integration.
streamer open a invoke AI instance on his discord server, bot takes custom token. token obtenable via a twitch point reward system by the viewers.

**Comment by ausbitbank** [comment]
I don't personally have any experience with the twitch points system and how to use via API, but I'd be open to adding it as an option if someone wants to write a function that integrates into the bots payment system.

As it is, if you can figure out a way to "burn"/remove channel points on the twitch side, you could manually give them credits using `!credits 123 @discordusername` . Will leave this open to explore further later
